docs/UserGuide.md

```diff
@@ -341,7 +370,7 @@ Example:
 
 ### Canceling a meeting: `meet`
 
-Schedules a meeting with the `NAME` given of the client with the `DATE` and `TIME` specified.
+Cancels current meeting with the `NAME` given of the client.
```

Comment:
This sentence seems a bit confusing. Maybe you can consider, `Cancels current meeting for the client with the specified NAME`